02-peer-protocol.md

```diff
@@ -11,9 +11,25 @@ The messages described in this document are grouped logically into 4 groups by t
  - Routing (types `256`-`511`): node and channel announcements, as well as any active route exploration.
 
 # Table of Contents
-
-FIXME
-
+  * [Channel](#channel)
```

Comment:
> Ah, I see you reverted that AFAICT? I'll squash and merge.

Yeah, I didn't want to mix two separate topics in the same PR. I am just OCDing on `funding_signed` vs `close_signature`  😵